csi/src/node/iscsiutil.rs

```diff
@@ -214,7 +232,7 @@ pub fn iscsi_detach_disk(device_path: &str) -> Result<(), String> {
     }
 }
 
-pub fn iscsi_find(uuid: &str) -> Option<String> {
+pub fn iscsi_find_(uuid: &str) -> Option<String> {
```

Comment:
I think it does not need to be public. Can we get some more meaningful name for it? In particular a name that would make clear the difference between iscsi_find and iscsi_find_. Comments for the functions (/// ...) would not make harm :-)

Reply:
Fixed by https://github.com/openebs/Mayastor/pull/232/commits/eb83c5027105004374de9d3eca8b798a7b8fd232